plugins/Proxy/Controller.php

```diff
@@ -91,43 +91,11 @@ public function redirect()
 				<br/><br/>You can access the page at: ' . $url);
         }
 
-        // mask visits to *.piwik.org
-        if (!self::isPiwikUrl($url)) {
-            Piwik::checkUserHasSomeViewAccess();
-        }
+        Piwik::checkUserHasSomeViewAccess();
```

Comment:
I was thinking it will be still good to have previous logic but makes sense to always check now.

Reply:
👍